Map.cpp

```diff
@@ -0,0 +1,85 @@
+#include <iostream>
+#include <string>
+#include <vector>
+#include <list>
+#include <utility>
+#include <sstream>
+#include <cctype>
+#include "FileManager.h"
+using namespace std;
```

Comment:
It's good practice to explicitly use std:: throughout ones program. I suggest you do so.